Follow Up : "A cool-looking box" page to be Semantically accurate (#15958)
mdn/content: files/en-us/learn/css/building_blocks/a_cool_looking_box/index.md

- A reasonable width for a large box, say around 200 pixels.
- A reasonable height for a large box, centering the text vertically in the process.

> - Horizontally Centered box.

Reviewer: I'm not sure what this means. Do we mean to say: "horizontally and vertically centred text?

Author: Text centering has been mentioned in the previous point itself, hence for the next point i added to center the <div> using margin:0 auto

- A slight increase in font size, to around 17-18 pixel computed style. Use rems. Write a comment about how you worked out the value.
- A base color for the design. Give the box this color as its background color.

> - A Contrasting color for the text; a black text shadow. Make it readable.

Reviewer: ```suggestion
- A contrasting color for the text and a black text shadow.
```
